libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1279,6 +1304,8 @@ bool ExpressionCompiler::visit(FunctionCall const& _functionCall)
 					utils().moveIntoStack(1, arguments.at(1)->annotation().type->sizeOnStack());
 				}
 			}
+			else
+				acceptAndConvert(*arguments.front(), *function.parameterTypes().front(), false);
```

Comment:
I could reduce the 3 occurrences of `accceptAndConvert` to 2, but that would then require duplicating some other stuff.